test/gui/shared/steps/steps.py

```diff
@@ -860,3 +904,123 @@ def step(context, itemType, resource):
         shutil.rmtree(resourcePath)
     else:
         raise Exception("No such item type for resource")
+
+
+@Given('the user has added the following server address:')
+def step(context):
+    newAccount = AccountConnectionWizard()
+    newAccount.addServer(context)
+    test.compare(
+        waitForObjectExists(
+            names.owncloudWizard_OwncloudHttpCredsPage_OCC_OwncloudHttpCredsPage
+        ).visible,
+        True,
+        "Assert credentials page is visible",
+    )
```

Comment:
Should we not include this type of code in a Page Object, if the selector `owncloudWizard_OwncloudHttpCredsPage_OCC_OwncloudHttpCredsPage` is for to check the visibility of a page? 🤔